test/jpypetest/test_jobject.py

```diff
@@ -166,3 +167,34 @@ def testObjectDouble(self):
         self.assertEqual(self.fixture.object_field, 2.6125)
         self.assertIsInstance(self.fixture.object_field,
                               JClass('java.lang.Double'))
+
+    def testObjectField(self):
+        with self.assertRaises(TypeError):
+            self.fixture.object_field = object()
+        with self.assertRaises(TypeError):
+            self.fixture.static_object_field = object()
+
+    def testArraySetRangeFail(self):
+        ja = JArray(JObject)(4)
+        with self.assertRaises(TypeError):
+            ja[:] = [ 1, 2, object(), 3]
```

Comment:
pep8 pls (extra space in beginning of list definition)

Reply:
No problem.  I will pep8 prior to merge.